scenariolib/event_pageview.go

```diff
@@ -46,6 +47,16 @@ func newViewEvent(e *JSONEvent) (*ViewEvent, error) {
 	}
 	event.clickRank = int(docNo)
 
+	if e.Arguments["pageViewField"] != nil {
+		if event.pageViewField, validcast = e.Arguments["pageViewField"].(string); !validcast {
+			return nil, errors.New("Parameter pageViewField must be of type string in ViewEvent")
+		}
+	} else if c.DefaultPageViewField != "" {
```

Comment:
There is a file called defaults.go
You should add that check there instead so all the defaults values are kept in one place and easier to find.